src/mon/FSCommands.cc

```diff
@@ -124,10 +124,10 @@ class FsNewHandler : public FileSystemCommandHandler
     string fs_name;
     cmd_getval(g_ceph_context, cmdmap, "fs_name", fs_name);
     if (fs_name.empty()) {
-        // Ensure fs name is not empty so that we can implement
-        // commmands that refer to FS by name in future.
-        ss << "Filesystem name may not be empty";
-        return -EINVAL;
+      // Ensure fs name is not empty so that we can implement
```

Comment:
Please revert this change, as it is not related to the purpose of this PR. If you need this change create a new PR, which may include similar changes in other files too.
Please refer: https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#4-separate-your-changes